bgbb/wrappers.py

```diff
@@ -77,23 +77,14 @@ def fit(self, df, **kw):
     @wraps(BetaGeoBetaBinomFitter._loglikelihood)
     def _loglikelihood(self, df, params=None, para=True):
         x, tx, T = unload(df, "frequency recency n")
-        params = mod_par_list(params, self.mod)
+        params = module_parameter_list(params, self.mod)
         return self.mod._loglikelihood(params, x, tx, T, para=para)
 
-    @wraps(BetaGeoBetaBinomFitter
-           .conditional_expected_number_of_purchases_up_to_time)
+    @wraps(BetaGeoBetaBinomFitter.conditional_expected_number_of_purchases_up_to_time)
     def cond_exp_rets_till(self, df, n_days_later, params=None, nb=False):
```

Comment:
conditional expected number of returns until n days later?

Reply:
Correct. It's conditioning on observed R/F/N patterns for an individual, assuming the whole population has the same params.

Reply:
This would be one of the functions I would recommend documenting, either through an un-shortened name or a docstring.